client/src/components/MainHeader.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import { PageHeader } from 'react-bootstrap';
+import '../css/style.css';
+
+/**
+ * Display a simple Header information.
+ */
+class MainHeader extends Component {
+	render() {
+		return (
+			<PageHeader bsClass="page-header">
+				Computer Systems Listing App
+				<br/><small>Simple UI for listing computer systems and their specifications.</small>
```

Comment:
First I would store the language support in the server side with all the needed translations for every language. Then, upon request I would return a JSON file with all translations for a specific language, with key values pairs that describe each string I would display in the UI.

On the client side I would create a top level translation component that would map the information received from the server with the needs for each component. So whenever a component is rendering it would request the translation component for a particular translation, with a specific key. The translator component just need to match that key with something received on the JSON file.

Of course we would need to have in the UI a way to select the supported languages.